jena-arq/src/main/java/org/apache/jena/riot/web/HttpNames.java

```diff
@@ -20,8 +20,8 @@
 
 public class HttpNames
 {
-    // Switch to org.apache.http.HttpHeaders
-    // or org.apache.jena.ext.com.google.common.net
+    // See also Guava com.google.common.net.HttpHeaders
```

Comment:
Maybe a little weird to put a Javadoc on a field, but maybe:
```
/**
* @see org.apache.jena.ext.com.google.common.net.HttpHeaders
```
?

Reply:
It's really an implementation note for the class. New line spacer added.